keyboards/treadstone32/keymaps/default/readme.md

```diff
@@ -0,0 +1,5 @@
+# Default keymap for treadstone32
```

Comment:
This file should be translated from the Japanese if possible, but if you need some time to do that it's okay. I just don't want it to be left behind.

Reply:
Maybe this readme not need english because noting to do. (Already understand by users.)